redaxo/src/core/lib/login/login.php

```diff
@@ -418,14 +418,14 @@ public static function startSession()
             );
 
             $started = rex_timer::measure(__METHOD__, static function () {
+                error_clear_last();
                 return @session_start();
             });
             if (!$started) {
-                $error = error_get_last();
-                if ($error) {
-                    rex_error_handler::handleError($error['type'], $error['message'], $error['file'], $error['line']);
+                if ($error = error_get_last()) {
+                    throw new rex_exception('Unable to start session: '.$error['message']);
```

Comment:
Vorher sind wir hier nur in unser normales Warning-Handling gegangen, ich denke aber, wir sollten immer in eine Exception gehen, wenn versucht wird, die Session zu starten, und das nicht klappt.

Reply:
Einige Nutzer berichten von Problem mit der Session. Das sollte man erst lösen, bevor hier Leute ein Update auf 5.9 machen und dann dadurch unkontrolliert Endnutzer mit Ooops-Meldungen konfrontiert werden.

Reply:
Wir haben gerade einen Über-100-Beiträge-langen Thread in Slack, wo wir zumindest von Oli und Skerbis die "Unable to start session"-Meldungen aus dem Log versuchen zu analysieren.

So sind wir auch zu diesem Nebenproblem gelangt. In die Zeile des Warning-Handlings konnte man vorher gar nicht kommen, wegen des fehlenden false in meinem anderen Kommentar.
Es ging also jetzt schon immer in die Exception, aber in die ohne Details aus der Warning.

Reply:
> Vorher sind wir hier nur in unser normales Warning-Handling gegangen, ich denke aber, wir sollten immer in eine Exception gehen, wenn versucht wird, die Session zu starten, und das nicht klappt.

sehe ich auch so.




> Wir haben gerade einen Über-100-Beiträge-langen Thread in Slack

OT: mittlerweile über 219 Antworten. :roll_eyes: